app/models/settings/user_experience.rb

```diff
@@ -21,7 +21,7 @@ class UserExperience < Base
     }
     # a non-public forem will redirect all unauthenticated pages to the registration page.
     # a public forem could have more fine-grained authentication (listings ar private etc.) in future
-    setting :public, type: :boolean, default: 0
+    setting :public, type: :boolean, default: 1
```

Comment:
By default we want the form to be public. Refer to [slack conversation](https://forem-team.slack.com/archives/C01S4QZUDNC/p1638896683273800).

Reply:
We should probably change this to `true` instead of `1`, usually we don't use numeric values for booleans in settings.